app/scripts/utils/date.ts

```diff
@@ -143,4 +151,26 @@ export function inputFormatToDate(inputFormat: string) {
   return parse(inputFormat, 'yyyy-MM-dd', new Date());
 }
 
-
+export type DateRangePreset =
+  | 'thisYear'
```

Comment:
instead of `thisYear`, `yearToDate` is a more common pattern, but as you said these are still TDB